Appsilon/box
# Get module exports

Issue #2 

## Description
* Adds function to retrieve exported functions of modules and packages.
* Supports all variations of `box::use()` calls

## How to test
This follows [Box contribution guidelines](https://klmr.me/box/articles/contributing.html#building)

* `make documentation` to build `NAMESPACE`
* `make test` to run unit tests
* `make check` to run checks.

CI was run [manually](https://github.com/Appsilon/box/actions).

## Definition of Done
- [ ] The change is thoroughly documented.
- [ ] The CI passes (`R CMD check`, linter, unit tests, spelling).
- [ ] Any generated files have been updated (e.g. `.Rd` files with `roxygen2::roxygenise()`)
